webpack.config.js

```diff
@@ -62,7 +63,11 @@ module.exports = {
         }
     },
     plugins: [
-        new WebpackDashDynamicImport()
+        new WebpackDashDynamicImport(),
+        new webpack.SourceMapDevToolPlugin({
+            filename: '[file].map',
+            exclude: ['async-plotlyjs']
```

Comment:
dash-bio doesn't have an `async-plotlyjs` chunk so I don't think this is doing anything... `NeedlePlot` and `AlignmentChart` both use `react-plotly.js`, looks like plotly.js is included in `async-alignment` and needleplot pulls it from there.

As long as it doesn't take too long to build the sourcemaps though, we can keep them all.

Reply:
Removed in 59586a1.